widget/text.go

```diff
@@ -284,7 +284,6 @@ func (r *textRenderer) Refresh() {
 
 		textCanvas.Alignment = r.provider.presenter.textAlign()
 		textCanvas.TextStyle = r.provider.presenter.textStyle()
-		textCanvas.Hidden = r.provider.Hidden
```

Comment:
This line does not seem like it should be removed - it's syncing with the owner state, not heirarchy like the other fixes.

Reply:
ack